bigvince/settings_.py

```diff
@@ -745,6 +745,9 @@ def get_secret(secret_arn):
 WEB_TITLE = "Vulnerability Notes Database"
 ORG_NAME = "Your Organization Name"
 CONTACT_EMAIL = "vuls@vince.org"
```

Comment:
We should use [RFC-2606](https://www.rfc-editor.org/rfc/rfc2606.html) compatible names for any dummy domain names even in sample configurations to avoid someone being able to register the domain.

suggestion: vuls@vince.example